docs/topics/email.txt

```diff
@@ -27,6 +27,39 @@ In two lines::
         fail_silently=False,
     )
 
+Use ``EmailMessage`` or ``EmailMultiAlternatives`` for additional functionality.
```

Comment:
```suggestion
Use :cls:`EmailMessage` or :cls:`EmailMultiAlternatives` for additional functionality.
```

(This will link to the documentation below.)